### DISCUSSION TITLE
[2024-06-24] Team GUI Demo Day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#10339] [Weekly Demo Day]
* #10244 - a small but important regression
* #10297 - GUI ought to request for dynamic config even if already got something from parent widgets, but did not handle the autoscoped constructors correctly. As a result, we missed drop-downs on some arguments.
* #10310 - on <kbd>ctrl</kbd>/<kbd>cmd</kbd> + output port hover.

[Screencast from 2024-06-18 17-06-59.webm](https://github.com/enso-org/enso/assets/3919101/4963eaa7-8b96-4fdb-af09-70de68c6df57)

* #10243 - there is only one issue with updating project name in dashboard, if the project was auto-opened on app start.

[Screencast from 2024-06-11 13-01-10.webm](https://github.com/enso-org/enso/assets/3919101/2917ad41-e080-482b-9a69-690e80087132)

* #10337 - in particular: the double arrow in drop down widgets.

**Comment by Frizi** [comment]
- Time-off for most of the iteration
- Implemented clickable link support in documentation editor https://github.com/enso-org/enso/pull/10353

https://github.com/enso-org/enso/assets/919491/b38f860a-ee4a-41a8-b936-41e9ef1602b9

https://github.com/enso-org/enso/assets/919491/059beca2-7737-41a1-a136-31a42ef9ee48

**Comment by kazcw** [comment]
Copy and paste:
- #10275 

https://github.com/enso-org/enso/assets/1047859/36ca3e6b-82b7-4004-9b4c-150c986ab363

Number widget:
- #10090
- #10234
- Display non-numeric initial values as placeholders: #9449

https://github.com/enso-org/enso/assets/1047859/bfe812e2-4d1c-477b-9500-f384fa499890

Output port:
- #10195

https://github.com/enso-org/enso/assets/1047859/90b6e67e-9036-4eb0-bc18-9550d610c923

Panning:
- #10280

https://github.com/enso-org/enso/assets/1047859/688ed136-4562-4c80-803c-14bbcdf1b231

Selection:
- Selection is now stable when an edit changes a node's ID: #10269 
- Debugged an issue and traced it to an open Chromium defect; workaround may be difficult and will be in a future iteration: #10312

**Comment by JaroslavTulach** [comment]
- faster startup (in cloud)
- TextEdit with custom IdMap #10283
- Async Ydoc initialization #10290
